portfolio/src/main/webapp/script.js

```diff
@@ -12,20 +12,14 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//fetching function
-// function getHelloContent(){
-//     fetch("/data").then(response => response.text()).then((data) =>{
-//         document.getElementById("server-data").innerText = data;
-//     })
-// }
+const maxLong =9223372036854775807;
```

Comment:
Avoid magic number. Use `Number.MAX_SAFE_INTEGER` instead if you really need one.